tests/test_time_series_plots.py

```diff
@@ -19,6 +20,9 @@
 pytestmark = pytest.mark.filterwarnings("ignore::UserWarning")
 os.environ["PYCARET_TESTING"] = "1"
 
+if sys.platform == "win32":
```

Comment:
Why are we skipping these on Windows?

Reply:
I think those are the corresponding references.

- https://github.com/pycaret/pycaret/pull/3814#issuecomment-1794003113
- https://github.com/ray-project/tune-sklearn/issues/283